bundles/org.openhab.binding.opensprinkler/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -163,7 +163,7 @@
 		<label>Current Draw</label>
 		<description>The current draw in mA</description>
 		<category>Energy</category>
-		<state readOnly="true"/>
+		<state readOnly="true" pattern="%.1f A"/>
```

Comment:
I believe you can safely use:
```suggestion
		<state readOnly="true" pattern="%.1f %unit%"/>
```

I'm not sure if you need to provide update instructions for this to be applied to existing managed things - see https://www.openhab.org/docs/developer/bindings/thing-xml.html#updating-thing-types. Did you test this?

Reply:
I do not see a way to update the state pattern by looking here:
https://www.openhab.org/schemas/update-description-1.0.0.xsd
Also from what I can guess, the state pattern is only used when the item is created?

Reply:
OK, I also did a quick test by changing the state pattern for a channel type when already having a managed Thing, and it seems thing update instructions are not needed for this scenario.

I did recently have a scenario in #16070 with changes in the channel type for some channels (`item-type` was changed), and here update instructions were needed and effective:
https://github.com/openhab/openhab-addons/blob/9872ca7789f488eea41cb16083eaf68f1f03eabf/bundles/org.openhab.binding.energidataservice/src/main/resources/OH-INF/update/instructions.xml#L32-L36